### DISCUSSION TITLE
Lots of external contribution in latest commits

### DISCUSSION THREAD

**Discussion by Schnabulation** [Jyers/Marlin#355] [Q&A] [answered]
Dear all

I have noticed there are a lot of commits from people other than @Jyers in the last days.
I really love the contribution and traction that this firmware gained in these days and a big thanks to @Jyers for his awesome work.
But I would like to know if I missed anything? Did this firmware get upstream support? Are these official Marlin devs? Or are these just a lot of pull requests that got accepted?

Not that I'm in any position to request anything, but I'm just wondering... (and I think there is no personal messaging system in GitHub so I can't ask @Jyers directly.)

**Comment by joeclifford** [comment, marked as answer]
Are you referring to commits such as this one https://github.com/Jyers/Marlin/commit/704b8cd83c507675d6f71621e1c35172779ac9d1

![image](https://user-images.githubusercontent.com/370866/113153458-06322580-922f-11eb-81b5-10a4d0092ca0.png)

If so, these are commits on the main Marlin repo. @Jyers merged branch bugfix-2.0.x into his fork to get all the latest changes. He did so in commit https://github.com/Jyers/Marlin/commit/ada1271af68bba74ff9196a750a4294900739ae3

**Reply by Schnabulation** [reply]
Oh shit, now I feel stupid - thank you Joe!

**Reply by joeclifford** [reply]
GitHub and GitHub desktop do a poor job of displaying the source of commits.

I prefer GitKraken, much more visual.

![image](https://user-images.githubusercontent.com/370866/113157126-8443fb80-9232-11eb-91d6-5657e49d8a78.png)

You can see all the commits from the bugfix-2.0.x branch in the 2nd column of the tree,

**Reply by Schnabulation** [reply]
Thank you. I actually work with Git Bash on my system and check the status directly on GitHub... I could have thought about the upstream commits because I see them as well in my fork (which is upstream-linked to Jyers fork), so...

Thanks again, man! Appreciate it!
